pkgs/development/python-modules/indexed-bzip2/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, isPy3k
+}:
+
+buildPythonPackage rec {
+  pname = "indexed_bzip2";
+  version = "1.5.0";
+  format = "setuptools";
+
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "b4a7fda1d69d7d0650609cfffef598a5e07df59f152e0fa110fbdf1075e07673";
```

Comment:
Could you update these hashes to the new format? e.g. `hash = "sha256-...";`

Reply:
This new hash format seems to have the downside that it cannot be simply copy-pasted from PyPI anymore. `echo 564393eae729a6fba0ed6964dc5a47d1a4aeabedd3b486a6453bb859a4f9c499 | xxd -r -p | base64` seems to work for conversion.